rates.md

```diff
@@ -16,9 +16,10 @@ module RATES
           <pot-pies> .Map      </pot-pies> // mapping (address => uint256) Address |-> Wad
           <pot-pie>  0:Wad     </pot-pie>
           <pot-dsr>  0:Ray     </pot-dsr>
-          <pot-chi>  0:Ray     </pot-chi>
+          <pot-chi>  0:Rat     </pot-chi> // arbitrary precision
           <pot-vow>  0:Address </pot-vow>
           <pot-rho>  0         </pot-rho>
+          <pot-live> false     </pot-live>
```

Comment:
The `constructor` for `Pot` starts it with `1`, so this should default to `true`.

Reply:
```suggestion
          <pot-live> true      </pot-live>
```

Reply:
@ehildenb do you want others to match the constructor? if so, `dsr` and `chi` should also default to`1`.

Reply:
Yes please.